service/experiment-requests.yaml

```diff
@@ -19,6 +19,16 @@
 # You can run "make presubmit" to do basic validation on this file.
 # Please add new experiment requests towards the top of this file.
 
+- experiment: 2020-12-23
+  description: "best of current state"
+  fuzzers:
```

Comment:
Do you want a regular coverage experiment or "type: bug" for bug based benchmarking.

Reply:
a regular with fries on the side please! :)